package.json

```diff
@@ -68,37 +69,36 @@
     "mobx": "^2.5.0 || ^3.0.0 || ^4.0.0 || ^5.0.0"
   },
   "devDependencies": {
+    "@types/chai": "^4.1.4",
+    "@types/lodash": "^4.14.114",
+    "@types/mocha": "^5.2.5",
+    "@types/node": "^10.5.3",
+    "@types/validatorjs": "^3.7.1",
     "ajv": "^5.2.2",
-    "babel-cli": "^6.26.0",
-    "babel-core": "6.23.1",
-    "babel-eslint": "8.0.2",
-    "babel-loader": "^7.1.4",
-    "babel-plugin-add-module-exports": "0.2.1",
-    "babel-plugin-lodash": "3.2.11",
-    "babel-plugin-transform-class-properties": "6.23.0",
-    "babel-plugin-transform-decorators-legacy": "1.3.4",
-    "babel-preset-es2015": "6.22.0",
-    "babel-preset-stage-0": "6.22.0",
-    "babel-register": "^6.26.0",
     "chai": "3.5.0",
     "codecov": "1.0.1",
     "commitizen": "2.9.6",
     "cross-env": "^5.1.4",
     "cz-conventional-changelog": "^2.0.0",
-    "eslint": "4.11.0",
-    "eslint-config-airbnb-base": "12.1.0",
-    "eslint-plugin-import": "2.8.0",
     "husky": "0.13.1",
     "json-loader": "0.5.4",
-    "mobx5": "npm:mobx@^5.0.0",
-    "mobx4": "npm:mobx@^4.0.0",
+    "mobx": "npm:mobx@^4.0.0",
```

Comment:
Mobx provides its own types so we need to have a way to get the types when we do a build. We do that here.